addons/website_sale_collect/models/delivery_carrier.py

```diff
@@ -47,6 +47,8 @@ def create(self, vals_list):
     def write(self, vals):
         if vals.get('delivery_type') == 'in_store':
             vals['integration_level'] = 'rate'
+            if not self.warehouse_ids:
+                vals['is_published'] = False
```

Comment:
Unexpected in a tests PR :thinking:
Probably should be in a separate FIX commit :eyes:

Reply:
Indeed, thanks!